lib/error.js

```diff
@@ -72,9 +72,15 @@ const ERRNO = {
   TOTP_REQUIRED: 160,
   RECOVERY_KEY_EXISTS: 161,
   UNKNOWN_CLIENT_ID: 162,
+  // XXX TODO: what happened to 163, is it up for grabs?
```

Comment:
No idea, it's not in the front end either.

Reply:
Fwiw:

https://github.com/mozilla/fxa-auth-server/pull/2742/files#diff-b83d27d3c5374e6d6de83271284db8c4R78

Reply:
HA!

Reply:
lol, I wonder if I originally had an additional error code in that PR.